logstash-core/src/main/java/org/logstash/ackedqueue/ext/JrubyAckedQueueMemoryExtLibrary.java

```diff
@@ -67,15 +66,11 @@ public IRubyObject ruby_initialize(ThreadContext context, IRubyObject[] args)
             int maxUnread = RubyFixnum.num2int(args[2]);
             long queueMaxBytes = RubyFixnum.num2long(args[3]);
 
-            Settings s = new MemorySettings(args[0].asJavaString());
             PageIOFactory pageIOFactory = (pageNum, size, path) -> new ByteBufferPageIO(pageNum, size, path);
             CheckpointIOFactory checkpointIOFactory = (source) -> new MemoryCheckpointIO(source);
-            s.setCapacity(capacity);
-            s.setMaxUnread(maxUnread);
-            s.setQueueMaxBytes(queueMaxBytes);
-            s.setElementIOFactory(pageIOFactory);
-            s.setCheckpointIOFactory(checkpointIOFactory);
-            s.setElementClass(Event.class);
+            Settings s = SettingsImpl.memorySettingsBuilder(args[0].asJavaString()).capacity(capacity)
+                .maxUnread(maxUnread).queueMaxBytes(queueMaxBytes).elementIOFactory(pageIOFactory)
+                .checkpointIOFactory(checkpointIOFactory).elementClass(Event.class).build();
```

Comment:
Do we have a preferred style for fluent/chained method calls - one call per line, or wrapping until line length boundaries?